src/sentry/monitors/logic/mark_failed.py

```diff
@@ -24,8 +24,17 @@
 
 def mark_failed(
     failed_checkin: MonitorCheckIn,
-    ts: datetime | None,
+    ts: datetime,
 ):
+    """
+    Given a failing check-in, mark the monitor environent as failed and trigger
```

Comment:
environment*

Reply:
Will clean up in a follow up if there's nothing else outstanding.

Reply:
yea looks good